nshlib/nsh_netcmds.c

```diff
@@ -377,8 +377,8 @@ static int nsh_foreach_netdev(nsh_netdev_callback_t callback,
   if (dir == NULL)
     {
       nsh_error(vtbl,
-                "nsh: %s: Could not open %s/net (is procfs mounted?): %d\n",
-                cmd, CONFIG_NSH_PROC_MOUNTPOINT, NSH_ERRNO);
```

Comment:
Why printing of NSH_ERRNO is removed?

Reply:
NSH_ERRNO may be a string or an integer:
https://github.com/apache/incubator-nuttx-apps/blob/master/nshlib/nsh.h#L434-L444
it's hard to keep it.

Reply:
Ok, I remove else keyword, so the errno will output too.